### PR TITLE
Implements pushing local folder to remote directory

### PR DESCRIPTION
**Pull request by alxberardi**

### CODE REVIEW

File: lib/pansophy/local/create_file.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+      def initialize(path, body)
+        @pathname = Pathname.new(path)
+        @body     = body
+      end
+
+      def call(options = {})
+        prevent_overwrite! unless options[:overwrite]
+        @pathname.dirname.mkpath
+        ::File.open(@pathname, 'w') do |f|
```
Comment (adz) [inline lib/pansophy/local/create_file.rb:14]: @pathname.write(@body)

**Comment by adz** [comment]
Looks good -- one minor nitpick..

**Comment by adz** [comment]
:+1:

File: lib/pansophy/local/create_file.rb (hunk 1 above)
Comment (alxberardi) [inline lib/pansophy/local/create_file.rb:14]: Unfortunately @adz, I had to use the old syntax for pre Ruby 2 (or 2.2, can't remember) compatibility.

**Pull request merged**